example/bin/main_raw.dart

```diff
@@ -26,6 +26,8 @@ void main() async {
       // the output file.
       const CliArg(name: 'map', value: "[comp_0_v]"),
       const CliArg(name: 'map', value: "[comp_0_a]"),
+      // Overwrite output files without asking
+      const CliArg(name: 'y'),
```

Comment:
We shouldn't overwrite output file without asking.

Reply:
I was just using -y as a proof of concept inside of the testing file to show that flags now work. I can remove it if you don't want it there by default.

Reply:
I'm not sure what you mean by "testing file"? It looks like this file is an entrypoint in `/bin`.

I think that anything a user might run, should prompt before overwriting a file. We don't want to be the reason that a user accidentally loses something.

Reply:
Yeah got it, it is just inside of the main_raw.dart file which the user doesn't necessarily interact with when using the library itself but I see the problems that could arise so removed the flag in the latest push.